.husky/pre-commit

```diff
@@ -1,4 +1,3 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn lint-staged
```

Comment:
please, return this line for the correct work of husky

Reply:
i already checked it, but forgot to push